src/usedrift.ts

```diff
@@ -0,0 +1,11 @@
+import { Drift } from './types.d'
+
+export const useDrift = (): Drift => {
+  const drift = window.drift
+
+  if (!drift) {
+    throw new Error("Drift hasn't been initialized yet.")
+  }
+
+  return drift as Drift
+}
```

Comment:
Hmm.. but this can fail unexpexted and you would always need to safeguard the call with Error checking.. hmmm.. what is the actual use case.. when do you need to get the drift object?

Reply:
only really when interacting with the widget or the API ( drift.api ).

Reply:
but still this can fail as it throws an error this can bring down the full app if not watched.. so you always need to catch it...  

i would suggest to return undefined | Drift

Reply:
changed in bb4af25